pkg/ip/ipmasq_linux.go

```diff
@@ -23,7 +23,9 @@ import (
 
 // SetupIPMasq installs iptables rules to masquerade traffic
 // coming from ip of ipn and going outside of ipn
-func SetupIPMasq(ipn *net.IPNet, chain string, comment string) error {
+func SetupIPMasq(ipn *net.IPNet, pluginName, containerID string) error {
```

Comment:
not a lot but seems this is used externally too https://grep.app/search?q=SetupIPMasq

Reply:
ah, right. Do I have to add a new API rather than changing the old one? I don't see anything about API stability guarantees...